_article-types/howto.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: article-type
+name: howto
```

Comment:
Jekyll provides the ``name`` attribute for you based on the filename, you don't need to add it here.

http://jekyllrb.com/docs/collections/#permalinks